bounties_api/std_bounties/models.py

```diff
@@ -179,14 +188,17 @@ class Bounty(BountyAbstract):
     def save(self, *args, **kwargs):
         fulfillment_amount = self.fulfillment_amount
         balance = self.balance
+        usd_price = self.usd_price
```

Comment:
It seems CircleCI is complaining because of this line. Was the intention to use this somewhere later in the save function?

Reply:
Nah hence I closed it, seems it is working without this line

Reply:
Well locally anyways, might be a cronjob that works locally but is stuck on the cluster?